.github/workflows/nudge.yml

```diff
@@ -15,3 +15,4 @@ jobs:
         uses: pavlovic-ivan/octo-nudge@v2
         with:
           webhooks: ${{ secrets.NUDGE_WEBHOOKS }}
+          conclusions: failure,cancelled
```

Comment:
Hi @pavlovic-ivan , it turns out that cancelled is not a valid conclusions value -> https://github.com/pavlovic-ivan/octo-nudge/blob/329060af1fb848e1ab0b3350dac3f5f5d9487eb9/util/index.js#L4. Is something that we could add to the nudge action or maybe it is not really possible to react on cancelled conclusions?

Reply:
Hi Marcin. `cancelled` is not currently supported by the `octo-nudgde`. However, i can add it and support it

Reply:
Hi @marcin-krystianc @jgiannuzzi . Please update the version of the octo-nudge action to `v3`. You should get the notifications for `cancelled` runs as well. Do let me know if it's not working (i've tested it, and it works in my setup).